docs/docs/get-started/setup-lightdash/install-lightdash.mdx

```diff
@@ -60,9 +60,8 @@ cd lightdash
 
 :::info
 
-If you're using a local dbt project, when you're asked `how you want to setup Lightdash`, select the option **"with local dbt"**
-and during the project creation keep the **Project directory** value as **"/usr/app/dbt"**.
-
+If you are using the install script, when you are asked **how do you want to setup Lightdash ?**, select the option  **with local dbt** and then provide the absolute path to your dbt project.
```

Comment:
Suggestion:

When using the install script, when you're asked `how do you want to setup Lightdash ?`, select the option  **with local dbt** and then provide the absolute path to your dbt project.